commons-mongo/src/main/scala/com/avsystem/commons/mongo/async/package.scala

```diff
@@ -0,0 +1,9 @@
+package com.avsystem.commons
+package mongo
+
+package object async {
```

Comment:
Can we do this in the same manner as all other implicit converter methods are implemented in `commons`?
That is, by a trait + companion object that extends it, like e.g. `SharedExtensions`.

This way users can choose between importing these conversions and mixing them in their project-level companion object as we do in most of our projects (at least in UMP-forks).